log/context.go

```diff
@@ -28,7 +28,7 @@ func (l *Logger) Fatal(args ...interface{}) {
 }
 
 func (l *Logger) Fatalln(args ...interface{}) {
-	glog.FatalDepth(1, l.ctx, args)
+	glog.FatalDepth(1, l.ctx, fmt.Sprintln(args...))
```

Comment:
Instead of turning this into a string here, should we just append it?

```
	glog.FatalDepth(1, append([]interface{}{l.ctx}, args...)...)
```

Reply:
Yes. `append` is a good way to handle `l.ctx`. Based on the above comments, my suggestion is:
```suggestion
        glog.FatalDepth(1, fmt.Sprintln(append([]interface{}{l.ctx}, args...)...))
```

Reply:
This sounds good.